conntrack.go

```diff
@@ -295,8 +295,9 @@ func ParseAttributes(logger *log.Logger, data []byte) (Con, error) {
 }
 
 // HookFunc is a function, that receives events from a Netlinkgroup.
+// The event group of the event is also passed to the hook function, to allow distinguishing between new, update, delete, etc.
 // Return something different than 0, to stop receiving messages.
-type HookFunc func(c Con) int
+type HookFunc func(c Con, table Table, eventgroup NetlinkGroup) int
```

Comment:
To not break the existing API I would prefer to have `table Table` and  `eventgroup NetlinkGroup` as part of `c Con`.

Reply:
For the table, moving it to Con is probably a good idea.
But the event type doesn't really belong to Con, because the same Con will have multiple events during its life. New, Update, Destroy, etc. This is more like a state of the object rather than a property of the object.